apps/projects/app/components/Panel/FundIssues/FundIssues.js

```diff
@@ -728,7 +729,13 @@ const IssueTitleBox = styled.div`
   grid-area: title;
   padding: ${2 * GU}px ${2 * GU}px 0 ${2 * GU}px;
   display: flex;
-  justify-content: space-between;
+  justify-content: flex-start;
+`
+const IssueAmountBox = styled.div`
+  grid-area: amount;
+  padding: ${2 * GU}px ${2 * GU}px 0 ${2 * GU}px;
```

Comment:
This might be easier to read as two rules:

```js
padding: ${2 * GU}px;
padding-bottom: 0;
```

However, I think maybe this logic would be cleaner if we moved it to the wrapping div.

```css
grid-gap: ${2 * GU}px;
padding: ${2 * GU}px;
```